pkg/util/kubeobjects/statefulset/consts.go

```diff
@@ -0,0 +1,7 @@
+package statefulset
+
+import "github.com/Dynatrace/dynatrace-operator/pkg/api"
+
+const (
+	AnnotationPvcHash = api.InternalFlagPrefix + "pvc-hash"
```

Comment:
```suggestion
	AnnotationPVCHash = api.InternalFlagPrefix + "pvc-hash"
```

I will ask so @andriisoldatenko doesn't have to 😅

Reply:
thanks!